.github/workflows/backend-checks.yml

```diff
@@ -29,6 +29,8 @@ jobs:
             ~/.cargo/git
             target
```

Comment:
Most changes are just formatting.  The only other changes are this here and others highlighted with comments below.